django/test/testcases.py

```diff
@@ -191,6 +192,21 @@ def __call__(self, result=None):
         set up. This means that user-defined Test Cases aren't required to
         include a call to super().setUp().
         """
+        self._setup_and_call(result, debug=False)
```

Comment:
Remove `debug=False` since that's the default.

Reply:
addressed in 3bf9e42